test/tests/parallel/set1/dovecot/mailbox_format_dbox.bats

```diff
@@ -30,11 +30,11 @@ function teardown() { _default_teardown ; }
   _wait_for_empty_mail_queue_in_container
 
   # Mail received should be stored as `u.1` (one file per message)
-  local MAILBOX_STORAGE='/var/mail/localhost.localdomain/user1/mailboxes/INBOX/dbox-Mails'
-  _count_files_in_directory_in_container "${MAILBOX_STORAGE}" 3
-  assert_output --partial "${MAILBOX_STORAGE}/dovecot.index.log"
-  assert_output --partial "${MAILBOX_STORAGE}/u.1"
-  assert_output --partial "${MAILBOX_STORAGE}/dovecot.index.cache"
+  local MAILBOX_STORAGE=
```

Comment:
Is `MAILBOX_STORAGE` used somewhere and I just don't see it?

Reply:
Nope, that's what I get for wrapping up a PR late at night, cut+paste the value, but I forgot to delete the rest of the line afterwards :)

Reply:
```suggestion
```